content/en/th3/500_accessibility/311_screenReaders.md

```diff
@@ -7,28 +7,64 @@ weight: '502'
 
 ## Reading with a screen reader
 
-When launching Thorium, if a screen reader is detected, the relative
-functions are activated.
+### Activating screen reader support 
 
-Note that screen readers use a separate buffer to access the HTML
-content which results in the visual scroll / page offset to be out of
-sync. In other words, Thorium is generally not aware of the user\'s
+Thorium automatically detects assistive technology, but, since 3.3, users must explicitly activate screen reader support by ticking the checkbox in the application settings (this is a global preference for the main library window as well as all reader windows). 
+
+In the previous versions, this support was automatically activated. We had to change because of third party apps creating false positives because they activate assistive technology privileges.
```

Comment:
> We had to change because of third party apps creating false positives because they activate assistive technology privileges.

==>

> We had to modify this feature because third-party applications can request "assistive technology" / "accessibility" privileges at the level of the operating system, which in turn causes Thorium to incorrectly detect screen readers.

Reply:
I used "The same proivilege as AT" becuase other wise that makes me want to opne issues on thos others apps to request them to respect AT privlege :)